src/listeners/AutoModerator.js

```diff
@@ -0,0 +1,20 @@
+const { EventListener } = require('../')
+
+module.exports = class MainListener extends EventListener {
+  constructor (client) {
+    super(client)
+    this.events = ['guildMemberAdd']
+  }
+
+  async onGuildMemberAdd (member) {
+    const guildDocument = this.database && await this.database.guilds.findOne(member.guild.id)
+    const t = this.i18next.getFixedT(guildDocument.language)
```

Comment:
You should define this inside the `if (guildDocument)`, otherwise it would cause errors (guildDocument.language is not possible if guildDocument is undefined)

Reply:
true.